src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/sensors/mouse.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
this is where we define the flow specific for each sensor. For example, mouse supports scroll listeners too or autoscroll.